packages/backend/server/src/base/graphql/logger-plugin.ts

```diff
@@ -21,9 +21,17 @@ export class GQLLoggerPlugin implements ApolloServerPlugin {
     ctx: GraphQLRequestContext<RequestContext>
   ): Promise<GraphQLRequestListener<GraphQLRequestContext<RequestContext>>> {
     const res = ctx.contextValue.req.res as Response;
-    const operation = ctx.request.operationName;
+    const info: Record<string, string | undefined> = {};
 
-    metrics.gql.counter('query_counter').add(1, { operation });
+    info.operation = ctx.request.operationName;
+
+    const headers = ctx.request.http?.headers;
+    if (headers) {
+      info.clientVersion = headers?.get('x-affine-version');
+      info.userAgent = headers?.get('user-agent');
```

Comment:
The value of userAgent is too much, adding it to the label will cause the metric calculation to be too large, so let's not add it for now

Reply:
removed